cypress-integration/integration/petition-clerk-verifies-auto-generated-nane.spec.js

```diff
@@ -0,0 +1,102 @@
+const {
```

Comment:
Out of curiosity, what was the reason for this being a cypress test over an integration test?

Reply:
The primary reason for making this test a cypress integration was because we were introducing new UI functionality (number of draft items) and wanted to write tests around the new UI implementation/flow.

cc: @halprin

Reply:
Hmmm, yes agreed, @Absolutestunna.  @rachelschneiderman, I could also be naive about when it is best to have a cypress test versus an integration test.  Is there a good rule of thumb?

Reply:
Good question

Reply:
Ok I totally get that! Thanks!

I think in general, I try to avoid adding Cypress tests as they are more expensive to write, run, and less reliable and more brittle than integration tests. Since most of the work done was abstracted into computeds and the `serveCaseToIrsInteractor` I think it could be tested with a jest integration test up until the point where it gets rendered on the page (which in the past we typically have not tested unless we are writing a smoketest/e2e test).